torch/jit/_recursive.py

```diff
@@ -117,19 +117,48 @@ def recursive_script(mod):
         if not _jit_internal.is_ignored_fn(mod.forward):
             methods = ('forward',)
     exported = []
+    overloads = []
     for name in dir(mod):
         item = getattr(mod, name)
         if callable(item):
             if _jit_internal.get_torchscript_modifier(item) is _jit_internal.FunctionModifiers.EXPORT:
                 exported.append(name)
+
+            # builtin functions like repr() in python 2 do not have __module__ defined
+            if hasattr(item, "__module__") and item.__module__ is not None:
+                method_overloads = _jit_internal._get_overloaded_methods(item, mod._get_name())
+                if method_overloads is not None:
+                    overloads.append((item, method_overloads))
+
     methods = methods + tuple(exported)
 
+    overload_name_mappings = dict(getattr(mod, "__overloads__", {}))
```

Comment:
Can you delete the `__overloads__` and replace it with these new decorators instead of adding both together? (or do that in a stacked PR)

Reply:
yea i was going to do that as a follow up, since that likely breaks the ScriptModules in quantization